compiler/rustc_feature/src/active.rs

```diff
@@ -150,7 +150,7 @@ declare_features! (
     (active, linkage, "1.0.0", Some(29603), None),
 
     /// Allows features specific to OIBIT (auto traits).
-    (active, optin_builtin_traits, "1.0.0", Some(13231), None),
+    (active, auto_traits, "1.0.0", Some(13231), None),
```

Comment:
Instead of simply renaming, you should add a new feature gate (with the current version number) and add `optin_builtin_traits` to `rustc_feature/src/removed.rs`. This allows you to warn the user that the feature gate has changed, cf.
https://github.com/rust-lang/rust/blob/1d2726726f8f3128e98191e4c6cb94bd76d0ddd4/compiler/rustc_feature/src/removed.rs#L72-L73
(Perhaps this advice should be added to the rustc guide if it's not there already.)

Reply:
Done. I'll open a dev-guide PR. Thanks for the help!